private/isalmostequal.m

```diff
@@ -110,7 +110,7 @@
     
     if ~isempty(abstol) && any(abs(a-b)>abstol)
       message{end+1} = sprintf('different values in %s', location);
-    elseif ~isempty(reltol) && any((abs(a-b)./(0.5*(a+b)))>reltol)
+    elseif ~isempty(reltol) && any((abs(a-b)./(0.5*abs(a+b)))>reltol)
```

Comment:
when a and/or b would be sufficiently negative, the difference would be negative and hence smaller than reltol